packages/webdoc-default-template/src/styles/scss-utils/README.md

```diff
@@ -0,0 +1,253 @@
+# scss-utils

```

Comment:
Add a link to the original source.

Reply:
> @Zekfad Would you mind making "code" have a max-width after which it scrolls horizontally?
> 
> <img alt="Screen Shot 2020-08-23 at 11 02 49 PM" width="1446" src="https://user-images.githubusercontent.com/22450567/90999493-c5857600-e594-11ea-8742-fcb06789164b.png">

What do you mean, whole center part is limited by sidebars as for now?

Reply:
Fixed in 4f21892457a08e5d7421f74348df333b77701dfe.